api.planx.uk/types.ts

```diff
@@ -12,3 +12,104 @@ export interface Flow {
     [key: string]: Node;
   };
 }
+
+export interface UserData {
+  answers?: Array<string>;
+  data?: Record<string, any>;
+  auto?: boolean;
+  feedback?: string;
+}
+
+export type Breadcrumb = Record<string, UserData>;
+
+export interface Team {
+  slug: string;
+  name: string;
+  domain?: string;
+  notifyPersonalisation: {
+    helpEmail: string;
+    helpPhone: string;
+    helpOpeningHours: string;
+    emailReplyToId: string;
+  };
+}
+
+export interface Passport {
+  data?: Record<string, any>;
+}
+
+export interface LowCalSession {
```

Comment:
Can you also please add `email: string;` & `flow_id: string;` to this type as they're required fields?